app/src/main/res/values-en/strings.xml

```diff
@@ -174,6 +174,12 @@
     <string name="hide_soft_when_physical_off_summary">Show soft keyboard when physical keyboard is
         used
     </string>
+    <string name="enable_alt_space_language_shortcut">Alt + Space switches languages</string>
```

Comment:
You don't need to specify the strings for `values-en/strings.xml`, crowd-in service generates those automatically (directly copies from `values/strings.xml`).